pkgs/tools/inputmethods/evscript/default.nix

```diff
@@ -7,11 +7,11 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
```

Comment:
Please set the version to unstable-XXXX-XX-XX where XXXX-XX-XX is the date of the commit.